__tests__/BsDotenv_test.bs.js

```diff
@@ -0,0 +1,20 @@
+// Generated by BUCKLESCRIPT VERSION 3.1.5, PLEASE EDIT WITH CARE
```

Comment:
Do we need to commit those files to run the test? I don't think so, but better to check.

Reply:
I added a pre hook to build the file. Initially I was thinking to do in the CI config file but good idea !